chapter06/mssw/exercise1.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/perl
+use v5.10;
+use strict;
+use warnings;
+use Storable qw( nstore retrieve );
+use Data::Dumper;
+
+require '/home/codio/workspace/chapter06/mssw/file_reader.pm';
```

Comment:
I'd probably have just created a `chapter06/lib/` directory and put it in there, then had a `use lib 'lib/file_reader.pm'` in my script.  Unless I'm missing some reason why it had to be a `require`.